Cubical/Data/Nat/Properties.agda

```diff
@@ -74,6 +74,15 @@ suc-predℕ (suc n) p = refl
 +-suc zero    n = refl
 +-suc (suc m) n = cong suc (+-suc m n)
 
++-one : ∀ m → m + 1 ≡ suc m
++-one zero = refl
++-one (suc m) = cong suc (+-one m)
+
++-two : ∀ m → m + 2 ≡ suc (suc m)
++-two zero = refl
++-two (suc m) = cong suc (+-two m)
```

Comment:
Is the particular way these are proved important? If not, they are special cases of `+-comm`.